zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RestErrorMapper.java

```diff
@@ -26,17 +27,37 @@
 
 public class RestErrorMapper {
 
+  public static final Function<BrokerRejection, ProblemDetail> DEFAULT_REJECTION_MAPPER =
+      rejection -> {
+        final String message =
+            String.format(
+                "Command '%s' rejected with code '%s': %s",
+                rejection.intent(), rejection.type(), rejection.reason());
+        final String title = rejection.type().name();
+        return switch (rejection.type()) {
+          case NOT_FOUND:
+            yield RestErrorMapper.createProblemDetail(HttpStatus.NOT_FOUND, message, title);
+          case INVALID_STATE:
+            yield RestErrorMapper.createProblemDetail(HttpStatus.CONFLICT, message, title);
+          case INVALID_ARGUMENT:
+          case ALREADY_EXISTS:
+            yield RestErrorMapper.createProblemDetail(HttpStatus.BAD_REQUEST, message, title);
+          default:
+            {
+              yield RestErrorMapper.createProblemDetail(
+                  HttpStatus.INTERNAL_SERVER_ERROR, message, title);
+            }
+        };
+      };
   private static final Logger REST_GATEWAY_LOGGER =
       LoggerFactory.getLogger("io.camunda.zeebe.gateway.rest");
```

Comment:
🙃  Just some heavy nitpicking: we can move the logger property at the top of the class just to get it out of the way. But this is just a matter of preference and patterns I've seen in the code.

Reply:
OK, yeah, I see what you mean. I will take this up on my list and simply adjust this with the next PR for the next job endpoints if you can live with that? 😄